internal/monitors/collectd/postgresql/postgresql.tmpl

```diff
@@ -0,0 +1,67 @@
+LoadPlugin postgresql
+
+<Plugin postgresql>
```

Comment:
You need to set the Interval at the plugin level too to match the `.IntervalSeconds` variable that is part of the monitor core config.